src/SOS/lldbplugin/CMakeLists.txt

```
@@ -42,66 +42,73 @@ if(NOT ENABLE_LLDBPLUGIN)
     return()
 endif()
 
+if(NOT "$ENV{LLDB_H}" STREQUAL "")
+    set(LLDB_H "$ENV{LLDB_H}")
+else()
+    # Glob matching LLDB headers in standard locations
+    file(GLOB LLDB_H_PATHS
+        "$ENV{ROOTFS_DIR}/usr/lib/llvm-*/include/lldb/API/LLDB.h"
+        "$ENV{ROOTFS_DIR}/usr/local/llvm*/include/lldb/API/LLDB.h"
+        "$ENV{ROOTFS_DIR}/usr/include/lldb/API/LLDB.h"
+    )
+
+    # Add explicitly specified path if provided
+    if(WITH_LLDB_INCLUDES)
+        file(GLOB EXTRA_LLDB_H_PATHS "${WITH_LLDB_INCLUDES}/lldb/API/LLDB.h")
+        list(APPEND LLDB_H_PATHS ${EXTRA_LLDB_H_PATHS})
+    endif()
+
+    # Sort the list to get the highest version last
+    list(SORT LLDB_H_PATHS)
```

```suggestion
    list(SORT LLDB_H_PATHS COMPARE NATURAL)
```
As I understood it the default sort is alphabetical which would sort versions < 10 after the versions > 10. I think we want a numerical sort instead?

---

Good points. I think current versions are double digits. llvm < 10 are very old and unlikely available on any official build system, so we will be fine with your natural sort approach.